backend/app/service/website_ssl.go

```diff
@@ -344,6 +370,8 @@ func (w WebsiteSSLService) ObtainSSL(apply request.WebsiteSSLApply) error {
 			}
 			printSSLLog(logger, "ApplyWebSiteSSLSuccess", nil, apply.DisableLog)
 		}
+
+		reloadSystemSSL(websiteSSL, logger)
 	}()
 
 	return nil
```

Comment:
I have not found any significant difference between the versions you provided. The current code appears to be well-maintained and adhering to best practices. However, if you need specific suggestions or optimizations based on current development standards and requirements, please share more details about those aspects you'd like to consider.